docs/game/cheat_sheet.md

```diff
@@ -4,10 +4,14 @@ Panellist Cheat Sheet
 Golden Rule!
 ------------
 
-Relax, take it easy and have fun.
+Relax, take it easy and have fun. The panel should have a generous, collegiate
+but competitive air.
 
-It's a little competitive, but ultimately we're all pulling together to make an
-enjoyable, entertaining and informative show. Winning isn't everything.
+Ultimately we're all pulling together to make an enjoyable, entertaining and 
+informative show. Think performance, rather than deathmatch.
+
+Trying too hard to win can lead to one voice dominating and variety is the spice
+of life. And this game. _Winning points isn't the point, or how you "win"._
```

Comment:
I mean — I'm with you 100% on the grammatical point.

However, it's for (comic) effect so it works...but it's very much Andy's voice and not yours so...